### PR TITLE
e2e: WithOffset -> GinkgoHelper

### PR DESCRIPTION
**Pull request by ffromani**
rather than use WithOffset variants, which composes badly, let's leverage ginkgo.GinkgoHelper() to achieve the same result in a more composable and easier way.

### CODE REVIEW

File: test/e2e/serial/tests/workload_placement_taint.go [hunk 1]
```diff
@@ -21,6 +21,7 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/onsi/ginkgo/v2"
```
Comment (shajmakh) [inline test/e2e/serial/tests/workload_placement_taint.go:24]: imported twice

Comment (ffromani) [reply test/e2e/serial/tests/workload_placement_taint.go]: fixed, thanks

**Pull request merged**